packages/lab/src/combo-box-next/useComboBox.tsx

```diff
@@ -0,0 +1,133 @@
+import { FocusEvent, KeyboardEvent, SyntheticEvent } from "react";
+import { useList, UseListProps } from "../list-next/useList";
+import { useComboboxPortal, UsePortalProps } from "./useComboboxPortal";
+
+interface UseComboBoxProps {
+  onBlur?: (event: FocusEvent<HTMLInputElement>) => void;
+  onFocus?: (event: FocusEvent<HTMLInputElement>) => void;
+  onMouseOver?: (event: SyntheticEvent) => void;
+  PortalProps?: UsePortalProps;
+  listProps: UseListProps;
+}
+
+export const useComboBox = ({
+  onFocus,
+  onBlur,
+  onMouseOver,
+  PortalProps,
+  listProps,
+}: UseComboBoxProps) => {
+  const {
+    open,
+    setOpen,
+    floating,
+    reference,
+    getPortalProps,
+    getTriggerProps,
+  } = useComboboxPortal(PortalProps);
+
+  const {
+    keyDownHandler: listKeyDownHandler,
+    activeDescendant,
+    focusVisibleRef,
+    selectedItem,
+    setSelectedItem,
+    setHighlightedItem,
+    highlightedItem,
+  } = useList({
+    ...listProps,
+  });
+
+  const focusHandler = (event: FocusEvent<HTMLInputElement>) => {
+    setOpen(true);
```

Comment:
think we no longer want this based on design review, keyboard focus shouldn't open list unless explicitly "Enter" or "Space"